src/improvements/tasks/MultisigTask.sol

```diff
@@ -323,17 +323,24 @@ abstract contract MultisigTask is Test, Script, StateOverrideManager {
     /// @notice Print the data to sign.
     function printEncodedTransactionData(bytes memory dataToSign) public pure {
         // logs required for using eip712sign binary to sign the data to sign with Ledger
-        console.log("\nData to sign:");
+        printTitle("DATA TO SIGN");
         console.log("vvvvvvvv");
         console.logBytes(dataToSign);
         console.log("^^^^^^^^\n");
 
-        console.log("---------- ATTENTION SIGNERS ----------");
+        printTitle("ATTENTION SIGNERS");
         console.log("Please verify that the 'Data to sign' displayed above matches:");
         console.log("1. The data shown in the Tenderly simulation.");
         console.log("2. The data shown on your hardware wallet.");
         console.log("This is a critical step. Do not skip this verification.");
-        console.log("---------------------------------------");
+    }
+
+    function printTitle(string memory title) private pure {
+        // forgefmt: disable-start
+        console.log("");
+        console.log(string.concat("\x1b[1m\x1b[36m", vm.toUppercase(title), "\x1b[0m"));
```

Comment:
I'd suggest using forge-std's `StdStyle.sol` library for readability

Reply:
huh, didn't know about this. I'll add it.

Reply:
https://github.com/ethereum-optimism/superchain-ops/pull/922/commits/7f0898394f0bea4d1f3979e97b1070577ac837d8